.vscode/extensions.json

```diff
@@ -2,7 +2,6 @@
   "recommendations": [
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
-    "jpoissonnier.vscode-styled-components",
-    "shinnn.stylelint"
```

Comment:
I guess we could remove styelint entirely from HSDS. From what I recall I had trouble with formatting styled-component file and I ended up with using stylehint to make it work correctly

We can always update the config too to make it better